ruleset/rules/0630-nextcloud_rules.xml

```diff
@@ -1,175 +1,163 @@
+    <description>NextCloud possible malicious request.</description>
+    <group>web,appsec,attack,pci_dss_6.5,pci_dss_11.4,gdpr_IV_35.7.d,nist_800_53_SA.11,nist_800_53_SI.4,tsc_CC6.6,tsc_CC7.1,tsc_CC8.1,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
   </rule>
 
-  <rule id="88203" level="10" frequency="8" timeframe="120">
-    <if_matched_sid>88202</if_matched_sid>
-    <description>NextCloud brute force (multiple failed logins).</description>
-    <group>authentication_failures,pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_11.4,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,nist_800_53_SI.4,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
+  <rule id="88205" level="8">
+    <if_sid>88200,88201</if_sid>
+    <field name="level">^4$</field>
     <options>no_full_log</options>
+    <description>NextCloud FATAL message.</description>
+    <group>gdpr_IV_35.7.d,</group>
   </rule>
 
-  <rule id="88213" level="6">
+  <rule id="88206" level="4">
     <if_sid>88200,88201</if_sid>
-    <match>File accessed: </match>
-    <description>NextCloud file accessed.</description>
-    <group>pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
+    <field name="level">^3$</field>
     <options>no_full_log</options>
+    <description>NextCloud ERROR message.</description>
   </rule>
 
-  <rule id="88214" level="6">
+  <rule id="88207" level="3">
     <if_sid>88200,88201</if_sid>
-    <match>File created: </match>
-    <description>NextCloud file created.</description>
-    <group>pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
+    <field name="level">^2$</field>
     <options>no_full_log</options>
+    <description>NextCloud WARN message.</description>
   </rule>
 
-  <rule id="88215" level="6">
+  <rule id="88208" level="2">
     <if_sid>88200,88201</if_sid>
-    <match>File deleted: </match>
-    <description>NextCloud file deleted.</description>
-    <group>pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
+    <field name="level">^1$</field>
     <options>no_full_log</options>
+    <description>NextCloud INFO message.</description>
   </rule>
 
-  <rule id="88216" level="6">
+  <rule id="88209" level="1">
     <if_sid>88200,88201</if_sid>
-    <match>Preview accessed: </match>
-    <description>NextCloud preview accessed.</description>
-    <group>pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
+    <field name="level">^0$</field>
     <options>no_full_log</options>
+    <description>NextCloud DEBUG message.</description>
   </rule>
 
-  <rule id="88204" level="6">
+  <rule id="88210" level="6">
```

Comment:
Is this rule's level correct?

Reply:
My guess is this should be a level 3. Not sure if modifying existing rule levels is in teh scope of the rework